link-grammar/linkage.c

```diff
@@ -876,7 +879,14 @@ void linkage_post_process(Linkage linkage, Postprocessor * postprocessor)
 	D_type_list * d;
 
 	if (NULL == linkage) return;
-	if (NULL == postprocessor) return;
+
+	if (NULL == postprocessor)
+	{
+		Sentence sent = linkage->sent;
+
+		sent->constituent_pp = post_process_new(sent->dict->hpsg_knowledge);
+		postprocessor = sent->constituent_pp;
```

Comment:
I'm confused by this and the next fix.  If the user did not ask for constituent printing, then we should not be doing any constituent processing. 

Keep in mind tht the constituent post-processing is different than the "regular" post-processing.